### ISSUE TITLE
Docs on "Ideal Capacity Limits" not clear

### ISSUE THREAD

**Issue by teameh** [Alamofire/AlamofireImage#330]
Great work on this lib and the clear and extensive docs!  🎉 👍 

There's one little thing that is not clear for me about the  "Ideal Capacity Limits" section:

> By default, the combination of caches offers the following storage capacities:
> 
> - 150 MB of on-disk storage
> - 20 MB of in-memory original image data storage
> - 100 MB of in-memory storage of filtered image content
> - 60 MB preferred memory capacity after purge of filtered image content

So for it's all clear, but the following note is not: 

> > If you do not use image filters, it is advised to set the memory capacity of the `URLCache` to zero to avoid storing the same content in-memory twice.

I wonder why would you set it to zero if you're not using filters. Users will always benefit from images being stored in memory if an image is retrieved from the cache right? Maybe you could add a little explanation to the docs here to make it more clear?

**Comment by teameh** [comment]
Hi @jshier thanks for elaborating.

> I think the docs are unclear as to where the different storage areas come from URLCache or AFI's cache. We'll update them.

That would be cool. An example on how to set the URLCache to zero might also be handy. I assumed something like this?

```
let config = ImageDownloader.defaultURLSessionConfiguration()
config.urlCache = URLCache(
  memoryCapacity: 20_000_000,
  diskCapacity: 150_000_000,
  diskPath: "com.my-sample-app"
)

let imageDownloader = ImageDownloader(
  configuration: config,
  downloadPrioritization: .fifo,
  maximumActiveDownloads: 16,
  imageCache: AutoPurgingImageCache()
)
```

**Comment by teameh** [comment]
ping @jshier ? :)

**Comment by teameh** [comment]
I can submit a PR for the docs containing this snippet if you could quickly check it?

**Comment by teameh** [comment]
Ping ping 😄 ?

**Comment by fiteroc** [comment]
I did this in AppDelegate and it is working. The app is using only the URLCache caching, not also the in-memory caching useful for images with filter.

        let imageCache = AutoPurgingImageCache(memoryCapacity: 0, preferredMemoryUsageAfterPurge: 0)
        UIImageView.af_sharedImageDownloader = ImageDownloader(imageCache: imageCache)

**Comment by cnoon** [comment]
Thanks for all the feedback here. I've just updated the documentation to be more clear for usage patterns for in-memory stores. I pushed ce2ea658 into `master` and it will be released as part of AFI 4.0.0 here shortly.

Thanks again! 🍻

**Comment by teameh** [comment]
Hi @cnoon thanks for adding that to the documentation. Would it be a good idea to add a code sample on how to do this? Is either [my example](https://github.com/Alamofire/AlamofireImage/issues/330#issuecomment-434329751) or @fiteroc 's [example](https://github.com/Alamofire/AlamofireImage/issues/330#issuecomment-543284773) the right way?
